owl/EASE-OBJ.owl

```diff
@@ -2564,7 +2574,9 @@
     <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Sharpness -->
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Sharpness">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Intrinsic"/>
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Geometry"/>
+        <rdfs:comment>The quality of having a thin edge or point that can cut something or make a hole in something.
```

Comment:
Seems like a good fir for the category :) thanks for adding the comment.
But you can also have a "sharp thought" or see a "sharp movie".
What about these cases? Is e.g. the sharpness of a movie also covered in this category?

Reply:
The comment related to sharpness is updated to only focus on the concept where it comes to sharp object. Rest of the concepts needs to be defined for future scenarios.